src/ServiceControl.Audit/Auditing/ImportFailedAudits.cs

```diff
@@ -24,7 +24,7 @@ public ImportFailedAudits(IDocumentStore store, AuditIngestor auditIngestor, Raw
         public Task Run(CancellationToken tokenSource)
         {
             source = tokenSource;
-            return Task.Run(() => Run<FailedAuditImport, FailedAuditImportIndex>(source));
+            return Run<FailedAuditImport, FailedAuditImportIndex>(source);
```

Comment:
@danielmarbach do you see any problem with this?

Reply:
It should be fine. I think we should also completely inline this method like we did in `ImportFailedErrors` plus remove the not needed dynamic casting if we are already touching this. Ok if I push here or do you want me to do another branch?

Reply:
Raised https://github.com/Particular/ServiceControl/pull/2416

Reply:
Thanks @danielmarbach. I'll merge this as soon as we've beaten TeamCity into submission, so that we have the GitHubSync update, and you can proceed with the more general fix in #2416.